kurbo/src/bezpath.rs

```diff
@@ -393,6 +393,17 @@ impl BezPath {
         cbox.unwrap_or_default()
     }
 
+    /// Get the last point of the path, if path is not empty.
```

Comment:
I think it's worth documenting here that the time complexity could be linear in the path length. I feel like this might be a little surprising otherwise, given that `Vec::last` is `O(1)`.